assets/components/checkboxInput/checkboxInput.jsx

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+
+// ----- Types ----- //
+
+type PropTypes = {
+  id: string,
+  onChange: (preference: boolean) => void,
+  labelText?: string,
+};
+
+
+// ----- Component ----- //
+
+export default function CheckboxInput(props: PropTypes) {
+  let label = '';
+
+  if (props.labelText) {
+    label = <label htmlFor={props.id}>{props.labelText}</label>;
+  }
+  return (
+    <div className={'component-checkbox'}>
+      <input
+        id={props.id}
+        type="checkbox"
+        onChange={e => props.onChange(e.target.checked)}
+      />
+      {label}
+    </div>
+  );
+}
+
+CheckboxInput.defaultProps = {
+  labelText: '',
```

Comment:
Should this be `null`? If it's not present, we don't use it at all.

Reply:
Since it is optional, it should have a value or `undefined`. Not `null`. More data [here](https://flow.org/en/docs/types/functions/)